ota-plus-web/docs/modules/ROOT/pages/updates-intro.adoc

```diff
@@ -1,20 +1,21 @@
-= What is an Update Configuration?
+= What is a software update?
 
-An update configuration is a set of instructions that you can later use in a campaign. These instructions specify the software versions that should be updated.
+A software update is a set of instructions that you can use in a campaign. These instructions specify the software versions that should be updated.
 
 image::img::workflow-update.png[image]
 
-Before you start creating update configuration, make sure that you've provisioned some target devices and uploaded your new software versions.
+=== Prerequisites
```

Comment:
```suggestion
== Prerequisites
```

We don't like out-of-order section levels; section levels are meant to carry semantic structure.

Reply:
Fixed